stable_baselines3/common/policies.py

```diff
@@ -547,11 +559,15 @@ def _build(self, lr_schedule: Schedule) -> None:
             # features_extractor/mlp values are
             # originally from openai/baselines (default gains/init_scales).
             module_gains = {
-                self.features_extractor: np.sqrt(2),
                 self.mlp_extractor: np.sqrt(2),
                 self.action_net: 0.01,
                 self.value_net: 1,
             }
+            if not self.share_features_extractor:
```

Comment:
Removing the negation and switching the blocks might make this easier to read. Same in line 587 below.

Reply:
Did it